pkg/validation/internal/bundle.go

```diff
@@ -17,8 +17,9 @@ var BundleValidator interfaces.Validator = interfaces.ValidatorFunc(validateBund
 
 // max_bundle_size is the maximum size of a bundle in bytes.
 // This ensures the bundle can be staged in a single ConfigMap by OLM during installation.
-// The value is derived from the standard upper bound for k8s resources (~4MB).
-const max_bundle_size = 4 << (10 * 2)
+// The value is derived from the standard upper bound for k8s resources (~1MB).
+// We will use this value to check the bundle compressed is < ~1MB
+const max_bundle_size = int64(1 << (10 * 2))
```

Comment:
If a bundle is really close to this size, but still under the limit, it _could_ still fail because there is overhead in the structure of configmap.

Does it make sense to go look at what one of these configmaps looks like on cluster and estimate what a reasonable overhead number of bytes is so we can reduce this maximum size accordingly?

Reply:
After we begin to compress I think will be very hard we face this issue. 
Also, I am not sure if we would really have this overhead scenario. 

WDYT about we address here the fix and then, if we check this scenario or as a follow-up check if would be required consider it?

Reply:
+1, this is an improvement we can make later.